packages/suite-web/e2e/support/utils/shortcuts.ts

```diff
@@ -47,8 +47,6 @@ export const passThroughBackupShamir = (shares: number, threshold: number) => {
 };
 
 export const passThroughInitMetadata = (provider: 'dropbox' | 'google') => {
-    cy.getConfirmActionOnDeviceModal();
```

Comment:
I wonder why this is not needed anymore. In order to avoid "enable labeling on device", don't you need to update test fixtures too? somewhere in https://github.com/trezor/trezor-suite/blob/1d321828d5817ba3f88aa7b746835b6598719786/packages/e2e-utils/src/mocks/dropbox.ts
also, are you plannig to add some tests covering this migration feature?

Reply:
I understand that the newly added test does not need this but I would expect old tests either to need this or to have some changes. maybe I don't remember it correctly. Anyway, this is tests, so feel free to dismiss this if tests are passing

Reply:
I have added changes to the old tests as well :)